lang/src/main/java/org/apache/shiro/lang/util/StringUtils.java

```diff
@@ -139,13 +139,16 @@ public static boolean startsWithIgnoreCase(String str, String prefix) {
      * is returned.
      *
      * @param in the input String to clean.
+     * @param trim specifies whether the input String should be trimmed or not
      * @return a populated-but-trimmed String or <code>null</code> otherwise
      */
-    public static String clean(String in) {
```

Comment:
Please re-target your PR against 1.9.x or 1.10.x (upcoming), so we can detect API changes via CI.
I see this is okay due to the following method, though.

Reply:
Here you go: https://github.com/apache/shiro/tree/1.10.x

Reply:
As requested, the PR has been re-targeted against 1.10.x.